Content.Shared/CCVar/CCVars.cs

```diff
@@ -67,13 +67,13 @@ public sealed class CCVars : CVars
         /// Ambience volume.
         /// </summary>
         public static readonly CVarDef<float> AmbienceVolume =
-            CVarDef.Create("ambience.volume", 0.50f, CVar.ARCHIVE | CVar.CLIENTONLY);
+            CVarDef.Create("ambience.volume", 1.5f, CVar.ARCHIVE | CVar.CLIENTONLY);
 
         /// <summary>
         /// Ambience music volume.
         /// </summary>
         public static readonly CVarDef<float> AmbientMusicVolume =
-            CVarDef.Create("ambience.music_volume", 0.50f, CVar.ARCHIVE | CVar.CLIENTONLY);
```

Comment:
Better to do 0.5 x the relevant multiplier IMO.

Reply:
huh

Reply:
ok well i opted to not do this because i dont want the range of the gain to be widened i just want it to be louder by default